src/Smalot/PdfParser/RawData/RawDataParser.php

```diff
@@ -455,59 +455,69 @@ protected function decodeXrefStream($pdfData, $startxref, $xref = [])
      *
      * @param string $pdfData  PDF data
      * @param array  $xref
-     * @param string $obj_ref  Object number and generation number separated by underscore character
+     * @param string $objRef   Object number and generation number separated by underscore character
      * @param int    $offset   Object offset
      * @param bool   $decoding If true decode streams
      *
      * @return array containing object data
      *
      * @throws Exception if invalid object reference found
      */
-    protected function getIndirectObject($pdfData, $xref, $obj_ref, $offset = 0, $decoding = true)
+    protected function getIndirectObject($pdfData, $xref, $objRef, $offset = 0, $decoding = true)
```

Comment:
Pls double check my refactorings

Reply:
Looks good to me. I didn't found any `$obj_ref` anymore.